src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -169,55 +160,21 @@ public function load(array $configs, ContainerBuilder $container)
     private function getBundleHierarchy(ContainerBuilder $container, array $config)
```

Comment:
This inner code can be moved out safely, there is not much logic to encapsulate now.

Reply:
Not sure, I prefer to keep it.

Reply:
Could we update the method name? Maybe `getBundleTemplatePaths()` or something like that?

Reply:
done